### PR TITLE
FKM-11 | CKeditor permissions for Editors.

### PR DESCRIPTION
**Pull request by sergey347**

### CODE REVIEW

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.cibox03.m1.propeople.com.ua/job/PR_BUILDER/35/

File: drupal/profiles/favrskov/modules/custom/favrskov_helper/favrskov_helper.install [hunk 1]
```diff
@@ -694,3 +694,10 @@ function favrskov_helper_update_7037() {
     'fields_shared',
   ));
 }
+
+/**
+ * Add permission.
+ */
+function favrskov_helper_update_7038() {
+  features_revert_module('config_user_roles');
+}
```
Comment (dporplenko) [inline drupal/profiles/favrskov/modules/custom/favrskov_helper/favrskov_helper.install]: EOL

**Comment by sergey347** [comment]
test this please

File: drupal/profiles/favrskov/modules/custom/favrskov_helper/favrskov_helper.install (hunk 1 above)
Comment (sergey347) [inline drupal/profiles/favrskov/modules/custom/favrskov_helper/favrskov_helper.install]: Bad luck)

**Pull request merged**